oat-sa/extension-tao-itemqti
# feat: add data-testid properties in the item authoring

**Related to:** https://oat-sa.atlassian.net/browse/AUT-774

**How to test:**
Check that `data-testid` attributes are added to Manage Items, Save, Preview, Add outcome declaration buttons on the Item Authoring page
